test/test_parallel.jl

```diff
@@ -208,17 +220,17 @@ import ParallelStencil.@gorgeousexpand
                         return
                     end
                     @parallel write_indices!(A);
-                    @test all(Array(A) .== [ix + (iy-1)*size(A,1) + (iz-1)*size(A,1)*size(A,2) for ix=1:size(A,1), iy=1:size(A,2), iz=1:size(A,3)])
+                    @test all(Array(A) .≈ [ix + (iy-1)*size(A,1) + (iz-1)*size(A,1)*size(A,2) for ix=1:size(A,1), iy=1:size(A,2), iz=1:size(A,3)])
```

Comment:
It would be good to achieve bitwise identical results. Maybe some casting using `$precision()` could do the trick

Reply:
Found it! There was two `memopt` tests that failed because the order of operations done to compute the reference array was not the same as the one for the kernel.
See 325defaa037f85bf632a5590abf13be34e972d59

Reply:
Great!